### ISSUE TITLE
PIF 17.9 not working with KernelSU and ZygiskNext

### ISSUE THREAD

**Issue by nimishkr** [chiteroman/PlayIntegrityFix#477]
Hello,
It might be after a recent ZygiskNext update (not sure/ but PIF 17.9 has started to fail Device integrity. I only get Basic integrity.
![Screenshot_20241113-130730_KernelSU~2](https://github.com/user-attachments/assets/9e91c031-3c2c-4dc9-95d2-55f8fe8a59c2)
![Screenshot_20241113-130734_KernelSU](https://github.com/user-attachments/assets/c7603e89-49b2-41ac-91a5-0a5a426bfd74)

**Comment by nimishkr** [comment]
Labels: [MEETS_BASIC_INTEGRITY]
Build fingerprint: OnePlus/NE2213/OP516FL1:14:user/release-keys
Brand: OnePlus
Device: OP516FL1
Model: NE2210
TestId: dGVzdGluZy04ZmM2MWMzNC0xMGVlLTQzMTQtOGRjNy00MDVhMjEzMjVhOTU=

**Comment by wary-hermit** [comment]
>It might be after a recent ZygiskNext update
Nope. I'm on Magisk beta channel, integrity was passing fine with Zygisk Next 1.2.2, 1.2.3, 1.2.4 versions. Recently only basic integrity started passing. Tried reverting to older Zygisk Next versions - same thing.

**Comment by M3551** [comment]
> > It might be after a recent ZygiskNext update
> 
> _Nope. I'm on Magisk beta channel, integrity was passing fine with Zygisk Next 1.2.2, 1.2.3, 1.2.4 versions. Recently only basic integrity started passing. Tried reverting to older Zygisk Next versions - same thing._
> 
> **Update**: Changed keybox to new leaked one - everything passes now. ![Screenshot_20241116-152954](https://private-user-images.githubusercontent.com/40537713/386848265-61b9fc6e-75bd-46d4-80ab-b2e8f1bb1f82.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE4NjYxNzMsIm5iZiI6MTczMTg2NTg3MywicGF0aCI6Ii80MDUzNzcxMy8zODY4NDgyNjUtNjFiOWZjNmUtNzViZC00NmQ0LTgwYWItYjJlOGYxYmIxZjgyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTE3VDE3NTExM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWMwYzg4NGQ4YzJhZDI3MzY0NDQxMGRkODg0OGE4MmUxZTFlNTY3YzI3MWU4ZWZjZDhhODg5ZjljMzE0OGIzYzYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.lqDchgt3FvQKZsQQMvUGXsz-2ZQJwRj3raoEKlF7nnA)

how to change to new keybox please help

**Comment by wary-hermit** [comment]
>how to change to new keybox please help  

First of all you're gonna need [Tricky Store](https://github.com/5ec1cff/TrickyStore) module. 
As for keybox files themselves - I get mine from Evolution X discord group, but I'm pretty sure they can be found being shared in other Android ROM/modding Telegram/Discord groups.

**Comment by M3551** [comment]
> > how to change to new keybox please help
> 
> First of all you're gonna need [Tricky Store](https://github.com/5ec1cff/TrickyStore) module. As for keybox files themselves - I get mine from Evolution X discord group, but I'm pretty sure they can be found being shared in other Android ROM/modding Telegram/Discord groups.

thanks, I already have tricky store but some how it doesn't work, I'd try do the tricky store's action again to see if it'd work
